paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlSchema.java

```diff
@@ -26,6 +26,7 @@
 import java.sql.DatabaseMetaData;
 import java.sql.ResultSet;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
```

Comment:
We can make this class pure, without `caseSensitive`.

Reply:
Done by: https://github.com/apache/incubator-paimon/pull/1123